scene/gui/rich_text_label.cpp

```diff
@@ -6173,6 +6334,20 @@ int RichTextLabel::get_total_glyph_count() const {
 	return tg;
 }
 
+int RichTextLabel::hit_test(const Point2 &coords) const {
+	ItemFrame *c_frame = nullptr;
+	Item *c_item = nullptr;
+	int c_line = 0;
+	int c_index = 0;
+	bool outside;
```

Comment:
Explicit may be good here
```suggestion
	bool outside = false;
```